File: post.php
<?php
    $entryData = array(
        'category' => 'Programming'
      , 'title'    => "This is PHP."
    );

    // This is our new stuff
    $context = new ZMQContext();
    $socket = $context->getSocket(ZMQ::SOCKET_PUSH, 'my pusher');
    $socket->connect("tcp://127.0.0.1:5555");

    $socket->send(json_encode($entryData));

File: post.py
import time
import zmq
import random

def post():
    context = zmq.Context()

    post_sender = context.socket(zmq.PUSH)
    post_sender.connect("tcp://127.0.0.1:5555")

    result = {'category' : 'Programming', 'title': 'This is Python.'}
    post_sender.send_json(result)

post()
